src/mockdata/mocks.ts

```diff
@@ -18,7 +19,7 @@ export const makeMetadata = () => {
     for (let i = 0; i < 10; i++) {
         allMetadata.push({
             id: faker.random.uuid(),
-            name: faker.name.title(),
+            name: DataSourceName.MET_API_FORECAST,
```

Comment:
Was this change necessary?

Reply:
Yes. 

I am, however, in the process of creating issues which will remove the need for this.

The reason for why they are there now is because the title is used to determine which query should be used to retrieve data from the backend. The type of the `name` field is no longer only a string, but actually an enum with the different names available in the DB.

This is error-prone, and thus I'm proposing a migration to using crafted IDs like `SSB_POPULATION` or `MET_API_FORECAST` for the metadata entries.

Also, refer to #83 -- though this will likely be made obsolete by the proposed changes.

Reply:
Sounds good and probably also smart